mkdocs_env.yml

```diff
@@ -0,0 +1,43 @@
+name: mkdocs
```

Comment:
I think we want our mkdocs environments to just be minimal, right? Not pinned versions of everything?

Reply:
Oh, oh! This shouldn't be here at all. I used to debug some version-drive misbehavior. Relevant packages are [here](https://github.com/openkinome/kinoml/pull/10/files/fb0c7342e6111f8d89ac245d9c147261945f34bb#diff-989fa9fc279181551c489ab45b745663R14). I still need to pin specific versions due to [this issue](https://github.com/pawamoy/mkdocstrings/issues/47), but that only affects `mkdocs`. I will remove all this dirt (and merge via squashing) before merge. Sorry!